Array Union/1_Array_Union.py

```diff
@@ -0,0 +1,28 @@
+def array_union(first,second):
+	try:
+		result = []
+		for number in first:
+			if number not in second:
```

Comment:
```if number not in second``` is standard python functionality.
let's change it to your own function to understand how it works
```if contains(second, number)``` where ```def contains```:
1. returns ```true``` if ```second``` contains ```number```
2. ```false``` if doesnt
3. not uses ```not in``` but regular ```for``` loop